inputs/manual_input.csv

```diff
@@ -317,4 +317,13 @@ offwind-float,investment,2050,1580,EUR/kWel,2020,https://doi.org/10.1016/j.adape
 offwind-float,FOM,2050,1.39,%/year,2020,https://doi.org/10.1016/j.adapen.2021.100067,
 offwind-float-station,investment,2030,400,EUR/kWel,2017,Haertel 2017; assuming one onshore and one offshore node + 13% learning reduction,
 offwind-float-connection-submarine,investment,2030,2000,EUR/MW/km,2014,DTU report based on Fig 34 of https://ec.europa.eu/energy/sites/ener/files/documents/2014_nsog_report.pdf,
-offwind-float-connection-underground,investment,2030,1000,EUR/MW/km,2017,Haertel 2017; average + 13% learning reduction,
\ No newline at end of file
+offwind-float-connection-underground,investment,2030,1000,EUR/MW/km,2017,Haertel 2017; average + 13% learning reduction,
+organic rankine cycle,lifetime,2020,30,years,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551",
+organic rankine cycle,efficiency,2020,0.12,per unit,2020,"Aghahosseini, Breyer 2020: From hot rock to useful energy: A global estimate of enhanced geothermal systems potential, https://www.sciencedirect.com/science/article/pii/S0306261920312551; Breede et al. 2015: Overcoming challenges in the classification of deep geothermal potential, https://eprints.gla.ac.uk/169585/","This is a rough estimate, depends on input temperature, implies ~150 C"
```

Comment:
Can you specify the efficiency more clearly? We've not done a good job there in the past. Either via the comment field (what is the input/output to which the efficiency refers to, probably heat to elec?) or via `X-input` and `Y-output`, see `Fischer-Tropsch` as an example.

Reply:
Makes sense, I hope I implemented this correctly!

Reply:
Yes,  Better! :)